tests/vehicle_efficiency.cpp

```diff
@@ -0,0 +1,114 @@
+#include "catch/catch.hpp"
+
+#include "game.h"
+#include "map.h"
+#include "map_iterator.h"
+#include "vehicle.h"
+#include "veh_type.h"
+#include "itype.h"
+#include "player.h"
+#include "cata_utility.h"
+#include "options.h"
+
+void clear_game( const ter_id &terrain )
+{
+    while( g->num_zombies() > 0 ) {
+        g->remove_zombie( 0 );
+    }
+
+    g->unload_npcs();
+
+    // Move player somewhere safe
+    g->u.setpos( tripoint( 0, 0, 0 ) );
+
+    for( const tripoint &p : g->m.points_in_rectangle( tripoint( 0, 0, 0 ),
+                                                       tripoint( MAPSIZE * SEEX, MAPSIZE * SEEY, 0 ) ) ) {
```

Comment:
Unrelated, but it may be worth adding an "all points on map" function to map class someday.
And/or "point in upper left" and "point in lower right" functions.